marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/relatedEntityMapping.spec.tsx

```diff
@@ -264,7 +264,7 @@ describe("Mapping", () => {
     mappingStepDetail.getRelatedEntityFromList("Relation (relatedTo Person)");
     cy.get("#entityContainer").scrollTo("bottom",  {ensureScrollable: false});
     mappingStepDetail.entityTitle("Person").should("not.be.visible");
-    browsePage.getPaginationPageSizeOptions().then(attr => {
+    cy.get(".ant-pagination-options .ant-select-selection-selected-value").then(attr => {
```

Comment:
I suggest not doing any "cy.get" inside spec files, is there a page object you can place it in?

Reply:
@csilva-ml  this was solved